northstar/Cargo.toml

```diff
@@ -45,8 +55,8 @@ proptest = "1.0"
 
 [features]
 default = []
-api = ["bytes", "derive-new", "npk", "serde_json", "tokio-util"]
-hello-world = []
+api = ["bytes", "derive-new", "npk", "serde_json", "tokio-util", "runtime"]
+npk = []
```

Comment:
The npk feature must have some deps that are not needed for the other features. e.g `which`